libbeat/beat/beat.go

```diff
@@ -104,7 +104,14 @@ func Run(name string, version string, bt Beater) {
 
 	// Runs beat inside a go process
 	go func() {
-		b.Start()
+		err := b.Start()
+
+		if err != nil {
+			// TODO: detect if logging was already fully setup or not
```

Comment:
I opened an issue here to solve this later: https://github.com/elastic/beats/issues/746